browser/src/slideshow/Transition2d.ts

```diff
@@ -148,6 +148,11 @@ class Transition2d extends TransitionBase {
 			gl.clear(gl.COLOR_BUFFER_BIT);
 		}
 
+		if (!this.program) {
```

Comment:
I don't understand why `program` would be null

Reply:
Me too, but I reported separate bug with debug info. This has to be fixed but also not break the app in the meantime